staging_service/globus.py

```diff
@@ -7,33 +7,37 @@
 
 def _get_authme_url():
     config = configparser.ConfigParser()
-    config.read(os.environ['KB_DEPLOYMENT_CONFIG'])
-    auth2_url = config['staging_service']['AUTH_URL']
+    config.read(os.environ["KB_DEPLOYMENT_CONFIG"])
```

Comment:
ran black ignore

Reply:
can we move these linter changes into a separate PR from the mapping one so it's easier to see and review what changed on actual implementation side?

Reply:
@leia-sefkin we can, but you can also just check the boxes to ignore these files in the PR review, as I provided of a listing of files to ignore

Reply:
we might as well also take a peek at em now too, it's pretty quick to review em